aea/aea.py

```diff
@@ -61,12 +60,10 @@ def __init__(
         self,
         identity: Identity,
         wallet: Wallet,
-        ledger_apis: LedgerApis,
```

Comment:
Ledger Apis are no longer exposed in the agent; they are now wrapped in the LedgerApiConnection